server/event_registration/admin.py

```diff
@@ -1,3 +1,22 @@
-# from django.contrib import admin
+from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin as BaseUserAdmin
+from .models import User
 
-# Register your models here.
+
+@admin.register(User)
+class UserAdmin(BaseUserAdmin):
+    ordering = ["email"]
+    list_display = ["email", "role", "is_staff", "is_active", "last_login"]
+    search_fields = ["email"]
+    fieldsets = (
+        (None, {"fields": ("email", "password")}),
+        ("Personal info", {"fields": ("role",)}),
+        ("Permissions", {"fields": ("is_active", "is_staff", "is_superuser", "groups", "user_permissions")}),
+        ("Important dates", {"fields": ("last_login", "created_at", "updated_at")}),
```

Comment:
```suggestion
        ("Important dates", {"fields": ("last_login", "created_at")}),
```

hey I tested this out, but when I create a new user via  the Django admin panel I get this error:
![image](https://github.com/user-attachments/assets/cc8f6e4e-6540-4570-a132-b90f58089817)

I fixed it by deleting `updated_at` from the UserAdmin fieldsets